### ISSUE TITLE
Patch doesn't work with ElAmigos release

### ISSUE THREAD

**Issue by 1232323123xasdasd123** [CookiePLMonster/SilentPatchScarface#3]
The patch doesn't seem to work with the elamigos release, I was wondering if you could update it for it to work with that release given that it's the most common you can obtain now a days, I had to resort to wiping out my old retail copy in order to install your patch and have the game actually launch.

The game also doesn't save after I install the patch, nor does it create the settings.ini.

Thank you in advance.

**Comment by CookiePLMonster** [comment]
Is this release 1.0? If so, I am aware of at least one incompatibility with 1.0 and will address it shortly (either today or tomorrow) via a hotfix.

If that's the case though, then I strongly suggest patching the game to 1.2.

**Comment by 1232323123xasdasd123** [comment]
Thanks man. Meanwhile I tried the retail release and fixed my saves by deleting the configuration folder VUGAMES on the registry, and restarting the game, then my game loaded right up, now the game runs and looks great, keep up the good work.

**Comment by CookiePLMonster** [comment]
Yeah, if you had to mess with registry that *does* sound like the issue I described. I'll need to look into that.
